modules/images/webp-uploads/load.php

```diff
@@ -573,3 +573,106 @@ function webp_uploads_update_featured_image( $html, $post_id, $attachment_id ) {
+	if ( ! is_array( $transforms ) ) {
+		return;
+	}
+
+	// We need to add fallback only if jpeg alternatives for the webp images are enabled for the server.
+	$jpegs = in_array( 'image/jpeg', $transforms['image/jpeg'], true ) && in_array( 'image/webp', $transforms['image/jpeg'], true );
+	$webp  = in_array( 'image/jpeg', $transforms['image/webp'], true );
+	if ( ! $jpegs && ! $webp ) {
+		return;
+	}
+
+	?>
+	<script>
+		( function() {
+			window.fallbackWebpImages = function( media ) {
+				for ( var i = 0; i < media.length; i++ ) {
+					try {
+						var ext = media[i].media_details.sources['image/jpeg'].file.match( /\.\w+$/i );
+						if ( ! ext || ! ext[0] ) {
+							continue;
+						}
+
+						var images = document.querySelectorAll( 'img.wp-image-' + media[i].id );
+						for ( var j = 0; j < images.length; j++ ) {
+							images[j].src = images[j].src.replace( /\.webp$/i, ext[0] );
+							var srcset = images[j].getAttribute( 'srcset' );
+							if ( srcset ) {
+								images[j].setAttribute( 'srcset', srcset.replace( /\.webp(\s)/ig, ext[0] + '$1' ) );
+							}
+						}
+					} catch ( e ) {
+					}
+				}
+			};
+
+			var loadMediaDetails = function( nodes ) {
+				var ids = [];
+				for ( var i = 0; i < nodes.length; i++ ) {
+					if ( nodes[i].nodeName !== "IMG" || ! nodes[i].src.match( /\.webp$/i ) ) {
+						continue;
+					}
+
+					var attachment = nodes[i].className.match( /wp-image-(\d+)/i );
+					if ( attachment && attachment[1] && ids.indexOf( attachment[1] ) === -1 ) {
+						ids.push( attachment[1] );
+					}
+				}
+
+				if ( ! ids.length ) {
+					return;
+				}
+
+				for ( var page = 0, pages = Math.ceil( ids.length / 100 ); page < pages; page++ ) {
+					var pageIds = [];
+					for ( var i = 0; i < 100 && i + page * 100 < ids.length; i++ ) {
+						pageIds.push( ids[ i + page * 100 ] );
+					}
+
+					var jsonp = document.createElement( 'script' );
+					jsonp.src = '<?php echo esc_js( get_rest_url() ); ?>wp/v2/media/?_fields=id,media_details&_jsonp=fallbackWebpImages&include=' + pageIds.join( ',' );
+					document.body.appendChild( jsonp );
+				}
+			};
+
+			var img = document.createElement( 'img' );
+
+			// Verify two webp images.
+			img.src = "data:image/webp;base64,UklGRjIAAABXRUJQVlA4ICYAAACyAgCdASoCAAEALmk0mk0iIiIiIgBoSygABc6zbAAA/v56QAAAAA==";			
+			img.onload = function() {
+				img.src = "data:image/webp;base64,UklGRh4AAABXRUJQVlA4TBEAAAAvAQAAAAfQ//73v/+BiOh/AAA=";
+			};
+
+			// Error handler will be executed if the browser doesn't support webp.
+			img.onerror = function() {
+				// Loop through already available images.
+				loadMediaDetails( document.querySelectorAll( 'img' ) );
+
+				// Start the mutation observer to update images added dynamically.
+				var observer = new MutationObserver( function( mutationList ) {
+					for ( var i = 0; i < mutationList.length; i++ ) {
+						loadMediaDetails( mutationList[i].addedNodes );
+					}
+				} );
+
+				observer.observe( document.documentElement, {
```

Comment:
I think we are only interested in the body right ?

```suggestion
				observer.observe( document.body, {
```

Reply:
Yep, updated.